### ISSUE TITLE
EVT_CHAR handler not called for TE_MULTILINE TextCtrl on macOS

### ISSUE THREAD

**Issue by ExplodingCabbage** [wxWidgets/Phoenix#804]
**Operating system**: macOS
**wxPython version**: 4.0.1
**Stock or custom build**: Stock
**Python version**: 3.6.1
**Stock or custom build**: Stock

**Description of the problem**: EVT_CHAR handlers on a TextCtrl are not called if the TextCtrl has the TE_MULTILINE Window Style flag set.

This has previously been noted by Hesky Fisher at https://stackoverflow.com/q/31620085/1709587, and I experienced the same thing today.

Demo: try running the program below, and observe that "Change handler 1 called" is printed if you type in the first TextCtrl, but nothing is printed if you type in the second TextCtrl.

```python
import wx

app = wx.App()
frame = wx.Frame(None, title="Test App", size=(640, 480))

panel = wx.Panel(frame)

text_ctrl_1 = wx.TextCtrl(panel, pos=(0, 0), size=(200, 25))
text_ctrl_2 = wx.TextCtrl(panel, pos=(0, 100), size=(200, 100),
                          style=wx.TE_MULTILINE)

def change_handler_1(event):
    print("Change handler 1 called")
    event.Skip()

def change_handler_2(event):
    print("Change handler 2 called")
    event.Skip()

text_ctrl_1.Bind(wx.EVT_CHAR, change_handler_1)
text_ctrl_2.Bind(wx.EVT_CHAR, change_handler_2)

frame.Show()
app.MainLoop()
```

**Comment by mesalu** [comment]
Yeah, I realized my mistake and went to clean up before I noticed your response. Ooops

**Comment by RobinD42** [comment]
This is an issue in wxWidgets 3.0, which appears to have already been fixed in 3.1. wxPython will get the fix in version 4.1, which is being worked on now.
